lib/animina_web/components/profile/profile_components.ex

```diff
@@ -598,15 +615,36 @@ defmodule AniminaWeb.ProfileComponents do
     ]
   end
 
+  defp active_or_inactive_mark(assigns) do
+    ~H"""
+    <%= if @state == @value do %>
+      <.active_mark language={@language} value={@value} />
+    <% else %>
+      <.inactive_mark language={@language} value={@value} />
+    <% end %>
+    """
+  end
+
+  defp inactive_mark(assigns) do
+    ~H"""
+    <div id={"inactive-mark-#{@value}"}>
+      <p class="mt-0.5 whitespace-nowrap rounded-md bg-gray-50 px-1.5 py-0.5 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10">
+        <%= with_locale(@language, fn -> %>
+          <%= gettext("Non Active") %>
```

Comment:
Please remove the `Non Active` pill.

Reply:
I have removed this, and we can also have them samw width